### ISSUE TITLE
AMD export is now a named module in v1.6.21

### ISSUE THREAD

**Issue by glenpike** [mbostock/topojson#274]
This is related to #269 and #267 from what I can see, but I will post here as a note that might help other people.

The new export mechanism generated by Rollup JS seems to have exported topojson as a named module.  This means you can no longer load the module via it's file path when you use it in AMD / RequireJS you have to create an alias, e.g.

v1.6.20
```
//This worked okay
define(['bower_components/topojson/topjson'], function(topojson) {
  console.log(topojson.feature);
})
```

v1.6.21
```
require.config({
  paths: {
    topojson: 'bower_components/topojson/topjson'
  }
});
```

```
define(['topojson'], function(topojson) {
  console.log(topojson.feature);
})
```

http://requirejs.org/docs/api.html#modulename
http://stackoverflow.com/questions/19720646/named-module-vs-unnamed-module-in-requirejs

**Comment by glenpike** [comment]
It might be possible to change the rollup command to export an anonymous module by not specifying the "-u" parameter - https://github.com/rollup/rollup/wiki/Command%20Line%20Interface

**Comment by mbostock** [comment]
This is a bug; I’ll remove the name.

**Comment by mbostock** [comment]
Fixed in 1.6.22.

**Comment by glenpike** [comment]
Nice, thanks, I'll give that a try next week :+1:
